src/psyclone/tests/dynamo0p3_test.py

```diff
@@ -5022,12 +5053,12 @@ def test_multiple_stencils():
         assert output5 in result
         if dist_mem:
             output6 = (
-                "      IF (f2_proxy%is_dirty(depth=f2_extent)) THEN\n"
-                "        CALL f2_proxy%halo_exchange(depth=f2_extent)\n"
+                "      IF (f2_proxy%is_dirty(depth=f2_extent+1)) THEN\n"
```

Comment:
How come the depths are now one greater than they were before? Did you fix a bug?